<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>北邮人论坛</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 244, 253);</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>971</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>16</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home">
     <attribute name="title">
      <string>首页</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>931</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="searchkind">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>发信人</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>标题</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchInfo">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchconfirm">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="listWidget_search">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>931</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Acadamics">
     <attribute name="title">
      <string>学术科技</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_aca">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>931</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Information">
     <attribute name="title">
      <string>信息社会</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>931</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Emotions">
     <attribute name="title">
      <string>情感天地</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_emo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>931</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>195</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="userInfo">
       <property name="text">
        <string>用户信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logout">
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>971</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="postcontent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>971</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendpost">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>190</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>发帖</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="posttitle">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>841</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>学术科技</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>信息社会</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>情感天地</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
